panel/src/components/Dialogs/Dialog.vue

```diff
@@ -186,16 +189,28 @@
   flex-direction: column;
 }
 
-.k-dialog-box[data-size="small"] {
-  width: 20rem;
+@media screen and (min-width: 20rem) {
+  .k-dialog-box[data-size="small"] {
+    width: 20rem;
+  }
 }
 
-.k-dialog-box[data-size="medium"] {
-  width: 30rem;
+@media screen and (min-width: 22rem) {
+  .k-dialog-box[data-size="default"] {
+    width: 22rem;
+  }
 }
 
-.k-dialog-box[data-size="large"] {
-  width: 40rem;
+@media screen and (min-width: 30rem) {
+  .k-dialog-box[data-size="medium"] {
+    width: 30rem;
+  }
+}
+
+@media screen and (min-width: 30rem) {
```

Comment:
I think you meant `min-width: 40rem`. Otherwise the 10rem are cut off.

Reply:
Yup! 😅